runner/report-exit.js

```diff
@@ -1,22 +1,20 @@
 const { generateDiscordCloudwatchFields, sendDiscordNotification, listECSTasks } = require('./helpers')
 
 async function main() {
-  if (process.env.AWS_ECS_CLUSTER.includes('prod')) {
-    // Only do this in prod because it's too noisy given the short interval of
-    // tnet and dev anchoring
-    const taskArns = await listECSTasks()
-    const fields = generateDiscordCloudwatchFields(taskArns)
-    const message = [
-        {
-        title: `CAS anchor task finished (${process.env.AWS_ECS_CLUSTER})`,
-        color: 3447003, // Blue
-        fields,
-        },
-    ]
-    const data = { embeds: message, username: 'cas-runner' }
-    const retryDelayMs = 300000 // 300k ms = 5 mins
-    sendDiscordNotification(process.env.DISCORD_WEBHOOK_URL_INFO_CAS, data, retryDelayMs)
-  }
+  // Only do this in prod because it's too noisy given the short interval of
+  // tnet and dev anchoring
+  const taskArns = await listECSTasks()
+  const fields = generateDiscordCloudwatchFields(taskArns)
+  const message = [
+      {
+      title: `CAS anchor task finished (${process.env.AWS_ECS_CLUSTER})`,
+      color: 3447003, // Blue
+      fields,
+      },
+  ]
+  const data = { embeds: message, username: 'cas-runner' }
+  const retryDelayMs = 300000 // 300k ms = 5 mins
+  sendDiscordNotification(process.env.DISCORD_WEBHOOK_URL_INFO_CAS, data, retryDelayMs)
```

Comment:
can we send the start and stop messages to a different channel based on env? Otherwise we're going to lose the ability to track the prod cas as it starts/stops while we drive through the request backlog, which was the whole reason we added these messages in the first place

Reply:
Yes they are different per environment already (under the hood)